content/api/cli.md

```diff
@@ -1,8 +1,264 @@
 ---
-title: Using the CLI
+title: CLI
 ---
-> Cli call
-> Cli options
-> --config
+## Overview
 
-> see also [[Using the configuration]]
+`webpack` provides a command line interface(cli) to configure and interact with your build.
```

Comment:
* Just webpack is enough since that's used in other docs too.
* command line interface(cli) -> Command Line Interface (CLI)